src/legacy/server/kbn_server.d.ts

```diff
@@ -19,10 +19,12 @@
 
 import { Server } from 'hapi';
 
-import { ConfigService } from '../../core/server/';
-import { ElasticsearchServiceSetup } from '../../core/server/';
-import { HttpServiceSetup } from '../../core/server/';
-import { PluginsServiceSetup } from '../../core/server/';
+import {
+  ElasticsearchServiceSetup,
+  HttpServiceSetup,
+  ConfigService,
+  PluginsServiceSetup,
+} from '../../core/server/';
```

Comment:
The trailing slash is unnecessary here on the import path. Please remove for consistency with most other imports.